storagemarket/lp2pimpl/net.go

```diff
@@ -31,9 +32,19 @@ var propLog = logging.Logger("boost-prop")
 const DealProtocolv120ID = "/fil/storage/mk/1.2.0"
 const DealProtocolv121ID = "/fil/storage/mk/1.2.1"
 const DealStatusV12ProtocolID = "/fil/storage/status/1.2.0"
+
+// The time limit to read a message from the client when the client opens a stream
 const providerReadDeadline = 10 * time.Second
+
+// The time limit to write a response to the client
 const providerWriteDeadline = 10 * time.Second
-const clientReadDeadline = 10 * time.Second
+
+// The time limit to wait for the provider to send a response to a client's request.
+// This includes the time it takes for the provider to process the request and
+// send a response.
+const clientReadDeadline = 60 * time.Second
```

Comment:
FYI: extrapolating from my logs I went for 90 in ♠️
Not advocating a change in boost, just sharing I've seen scary times in an SP log

Reply:
Ok thanks, that's good to know.
I think separately we should look into why third party tools are taking a long time to respond.